### PR TITLE
Home

### PR DESCRIPTION
**Pull request by santhoshpemmaka**
Added: Home page to the E-commerce Page. Please give any suggestion for improvements

### CODE REVIEW

File: checkout.css [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+    width: 100%;
+    justify-content: space-between;
+    margin-bottom: 1rem;
+    font-size: 1.4rem;
+    color: #282c3f;
+    font-weight: 400;
+}
+
+.card-cost-discount{
+    color: #03a685 !important;
```
Comment (itsmebipul00) [inline checkout.css:158]: Arranging codes or code files to maintain specificity could be done to avoid using !important

File: header.html [hunk 2]
```diff
@@ -124,7 +100,7 @@
 						<input type="radio" class="category-checkbox" />
 						<label class="category-text">3 stars & above</label>
 					</div>
-                    <div class="nav-options">
+					<div class="nav-options">
 						<input type="radio" class="category-checkbox" />
```
Comment (itsmebipul00) [inline header.html:104]: id and for in input and label could be used

File: home.css [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+	margin: 1rem 0 1rem 2rem;
+	text-transform: uppercase;
+	color: var(--primary-text-color);
+	font-weight: 500;
+	padding: 0.5rem 1rem;
+}
+.home-layout {
+	display: grid;
+	margin-top: 3rem;
+	grid-template-columns: 1fr 1fr 1fr 1fr 1fr;
```
Comment (itsmebipul00) [inline home.css:44]: repeat(5, 1fr)

File: home.css [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+	height: 2rem;
+	margin: 0 1.5rem;
+}
+
+@media only screen and (max-width: 1024px) {
+	:root {
+		font-size: 50%;
+	}
+	.home-layout {
+		grid-template-columns: 1fr 1fr 1fr;
```
Comment (itsmebipul00) [inline home.css:94]: repeat(3, 1fr)

**Review by itsmebipul00 — APPROVED** [review]
Good to merge, if possible with changes

File: checkout.css [hunk 5]
```diff
@@ -0,0 +1,186 @@
...
+}
+.container {
+	padding: 3rem 0;
+	text-align: center;
+}
+.checkout-heading {
+	display: inline-block;
+	font-size: 2rem;
+	font-weight: 700;
+	color: #282c3f;
```
Comment (Ranjeetk489) [inline checkout.css:27]: use var

File: checkout.css [hunk 6]
```diff
@@ -0,0 +1,186 @@
...
+}
+.card-item {
+	display: flex;
+	width: 100%;
+	border: 2px solid var(--border-color);
+	border-radius: 0.35rem;
+	margin: 1rem 0;
+}
+.image-container {
+	width: 157px;
```
Comment (Ranjeetk489) [inline checkout.css:48]: use rem

File: checkout.css [hunk 7]
```diff
@@ -0,0 +1,186 @@
...
+}
+.card-item-heading {
+	font-size: 1.6rem;
+	font-weight: 500;
+	color: #282c3f;
+	margin-bottom: 0.3rem;
+}
+.card-item-desc {
+	font-size: 1.2rem;
+	margin-bottom: 0.3rem;
```
Comment (Ranjeetk489) [inline checkout.css:70]: use px

File: checkout.css [hunk 8]
```diff
@@ -0,0 +1,186 @@
...
+}
+.card-item-pricecost {
+	font-weight: 700;
+	font-size: 1.6rem;
+	color: #282c3f;
+	margin-right: 1rem;
+}
+.card-item-actualcost {
+	font-size: 1.6rem;
+	color: #282c3f;
```
Comment (Ranjeetk489) [inline checkout.css:89]: use var

**Review by Ranjeetk489 — APPROVED** [review]
after implementing said changes by me and others code will be good to merge

File: checkout.css (hunk 1 above)
Comment (santhoshpemmaka) [reply checkout.css:158]: I removed important word color property. thanks :)

File: home.css (hunk 3 above)
Comment (santhoshpemmaka) [reply home.css:44]: i made changes

File: home.css (hunk 4 above)
Comment (santhoshpemmaka) [reply home.css:94]: i made changes

File: checkout.css (hunk 6 above)
Comment (santhoshpemmaka) [reply checkout.css:48]: i made changes. thanks :)

File: checkout.css (hunk 7 above)
Comment (santhoshpemmaka) [reply checkout.css:70]: Actually i'm following same format other files. I will be change later.

File: checkout.css (hunk 8 above)
Comment (santhoshpemmaka) [reply checkout.css:89]: i made changes.

File: checkout.css (hunk 5 above)
Comment (santhoshpemmaka) [reply checkout.css:27]: i made changes.

**Comment by santhoshpemmaka** [comment]
I resolve all issues. I'm merged with the developer branch. 👍

**Pull request merged**